saleor/graphql/account/filters.py

```diff
@@ -81,12 +83,17 @@ class Meta:
 
 class PermissionGroupFilter(django_filters.FilterSet):
     search = django_filters.CharFilter(method=filter_search)
+    ids = GlobalIDMultipleChoiceFilter(method=filter_by_id(account_types.Group))
 
 
 class StaffUserFilter(django_filters.FilterSet):
     status = EnumFilter(input_class=StaffMemberStatus, method=filter_staff_status)
     search = django_filters.CharFilter(method=filter_user_search)
-
+    ids = GlobalIDMultipleChoiceFilter(
```

Comment:
Maybe I am wrong but I would say that it doesn't allow you to return `staff users` but the all users. You don't validate anywhere that given ID is a `staff user`.

Reply:
It's a mean of filtering the existing list, rather than fetching the base queryset. `filter_user_search` is also shared between "customers" and "staffUsers" queries